mdn/kuma
# Fix window not defined error on /payments

Hey @peterbe, I had some time so I wanted to see if I could help! Let me know what you think of this solution. 

**Changes:** 
- Pass `settings.CONTRIBUTION_SUPPORT_EMAIL` as data in the Jinja template
- Access email as a prop in PaymentsIndex

**To test:**
Go to /payments, check that the `window is not defined` error is gone in the logs. 

Part of #6790
